crates/bevy_ui/src/lib.rs

```diff
@@ -226,7 +228,8 @@ fn build_text_interop(app: &mut App) {
                 .in_set(UiSystem::PostLayout)
                 .after(bevy_text::remove_dropped_font_atlas_sets)
                 // Text2d and bevy_ui text are entirely on separate entities
-                .ambiguous_with(bevy_text::update_text2d_layout),
+                .ambiguous_with(bevy_text::update_text2d_layout)
+                .ambiguous_with(bevy_text::calculate_bounds_text2d),
```

Comment:
Maybe these two should be in a single set to ignore?

Reply:
Possibly: I wanted to keep the changes relatively minimal though.